csp_billing_adapter/bill_utils.py

```diff
@@ -113,9 +133,9 @@ def get_billable_usage(
 
     for metric, data in config.usage_metrics.items():
         if data['usage_aggregation'] == 'average':
-            usage = get_average_usage(metric, usage_records)
+            usage = get_average_usage(metric, usage_records, bill_time)
         elif data['usage_aggregation'] == 'maximum':
-            usage = get_max_usage(metric, usage_records)
+            usage = get_max_usage(metric, usage_records, bill_time)
 
```

Comment:
The README.md in examples mentions three possible settings for usage_aggregation, maximum, average, current. Does this work if it is set to current?

Reply:
I do not see any tests for "current" so if the is not valid anymore we should update the README.md. If it is valid do we need tests for it?

Reply:
Current is not implemented. It's a possible future feature and not in the scope of v1.

Reply:
that is fine but then you need to update the README.md with such.

Reply:
whatever suits you. I gave my opinion on why I think it is important to be consistent

Reply:
https://github.com/SUSE-Enceladus/csp-billing-adapter/pull/61